build_config.rb

```diff
@@ -26,6 +26,7 @@
 
   # ngx_mruby extended class
   conf.gem './mrbgems/ngx_mruby_mrblib'
+  conf.gem :github => 'ksss/mruby-stringio' # required by rack-based-api
```

Comment:
`ksss/mruby-stringio` was linked to ngx_mruby if linking `rack-based-api`. So, maybe we resolve this problem by this dependency configuration, https://github.com/matsumoto-r/ngx_mruby/pull/178#issuecomment-227367590